packages/strapi-design-system/src/Typography/Typography.stories.mdx

```diff
@@ -22,54 +22,63 @@ This is the doc of the different Text / Typography components.
-        <Typography variant="epsilon">Epsilon - subtitle</Typography>
-      </Box>
-      <Box paddingTop={2} paddingLeft={4}>
-        <Typography variant="omega">Omega - body</Typography>
-      </Box>
-      <Box paddingTop={2} paddingLeft={4}>
-        <Typography variant="omega" fontWeight="semiBold">
-          Omega semibold - body highlight
-        </Typography>
-      </Box>
-      <Box paddingTop={2} paddingLeft={4}>
-        <Typography variant="omega" fontWeight="bold">
-          Omega bold - button text
-        </Typography>
-      </Box>
-      <Box paddingTop={2} paddingLeft={4}>
-        <Typography variant="pi">Pi - small text</Typography>
-      </Box>
-      <Box paddingTop={2} paddingLeft={4}>
-        <Typography variant="pi" fontWeight="bold">
-          Pi bold - small button text
-        </Typography>
-      </Box>
-      <Box paddingTop={2} paddingLeft={4}>
-        <Typography variant="sigma">Sigma - table label</Typography>
-      </Box>
-      <Box paddingTop={2} paddingLeft={4} width="200px">
-        <Typography ellipsis>Ellipsis example - long text - long text</Typography>
-      </Box>
+    <Box paddingTop={2} paddingLeft={4}>
+      <Typography variant="alpha">Alpha - header 1</Typography>
+    </Box>
+    <Box paddingTop={2} paddingLeft={4}>
+      <Typography variant="beta">Beta - header 2</Typography>
+    </Box>
+    <Box paddingTop={2} paddingLeft={4}>
+      <Typography variant="delta">Delta - header 3</Typography>
+    </Box>
+    <Box paddingTop={2} paddingLeft={4}>
+      <Typography variant="epsilon">Epsilon - subtitle</Typography>
+    </Box>
+    <Box paddingTop={2} paddingLeft={4}>
+      <Typography variant="omega">Omega - body</Typography>
+    </Box>
+    <Box paddingTop={2} paddingLeft={4}>
+      <Typography variant="omega" fontWeight="semiBold">
+        Omega semibold - body highlight
+      </Typography>
+    </Box>
+    <Box paddingTop={2} paddingLeft={4}>
+      <Typography variant="omega" fontWeight="bold">
+        Omega bold - button text
+      </Typography>
+    </Box>
+    <Box paddingTop={2} paddingLeft={4}>
+      <Typography variant="pi">Pi - small text</Typography>
+    </Box>
+    <Box paddingTop={2} paddingLeft={4}>
+      <Typography variant="pi" fontWeight="bold">
+        Pi bold - small button text
+      </Typography>
+    </Box>
+    <Box paddingTop={2} paddingLeft={4}>
+      <Typography variant="sigma">Sigma - table label</Typography>
+    </Box>
+    <Box paddingTop={2} paddingLeft={4} width="200px">
+      <Typography ellipsis>Ellipsis example - long text - long text</Typography>
+    </Box>
+  </Story>
+</Canvas>
+
+### Ellipsis
+
+By using the prop `ellipsis` it is possible to limit the text to one line and to
+signal to the user there is more to it.
+
+<Canvas>
+  <Story name="ellipsis">
+    <Box maxWidth="150px" borderStyle="solid" borderWidth="1px" borderColor="neutral200">
+      <Typography ellipsis>Loong text, that will be cut off</Typography>
```

Comment:
Can you also add the Props table with `ArgTable` at the end of the mdx file please